### PR TITLE
Fix Broken Link to guides

### PR DESCRIPTION
**Pull request by whatnotery**
Hey just wanted to fix a link in the docs to the getting started guide. It should work now. Let me know if any changes are needed

### CODE REVIEW

**Comment by wuarmin** [comment]
The current link of the getting-started guide is https://guides.hanamirb.org/v2.0/introduction/getting-started

File: source/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ API documentation for [Hanami](http://hanamirb.org).
     cd bookshelf && bundle
     bundle exec hanami server # visit http://localhost:2300
 
-Please follow along with the [Getting Started guide](http://hanamirb.org/guides/1.2/getting-started).
+Please follow along with the [Getting Started guide](https://guides.hanamirb.org/v1.3/introduction/getting-started/).
```
Comment (wuarmin) [inline source/README.md:20]: https://guides.hanamirb.org/v2.0/introduction/getting-started

**Review by wuarmin — CHANGES_REQUESTED** [review]

File: source/README.md (hunk 1 above)
Comment (whatnotery) [reply source/README.md:20]: Just made that change!

**Review by wuarmin — APPROVED** [review]

**Comment by whatnotery** [comment]
Hey @wuarmin what would it take to get this merged into main now that I've made the appropriate changes?

**Comment by wuarmin** [comment]
@whatnotery I'm not an official maintainer. I just reviewed your PR. I think you have to be a little patient. If the change is wanted, it will also be merged.

**Comment by whatnotery** [comment]
@wuarmin ahh I see, hopefully it'll get approved. the current link 404s :( the only other pull request in this repo is also for a broken link and it's 4 years old and unmerged so I guess we'll see

**Comment by cllns** [comment]
Thanks! I tweaked it to be the version of the link that redirects to the latest version

**Pull request merged**